Week01/Problem04/klundquist_04.py

```diff
@@ -0,0 +1,17 @@
+
+#Largest Palindrome Product
+i = 0
+b = True
+while i <= 100 and b:
+    j = 0
+    while j<= 100 and b: 
+        num1 = 999 - i
+        num2 = 999 - j
+        res = num1 * num2
+        res_str = str(res)
+        if res_str[0] == res_str[5] and res_str[1] == res_str[4] and res_str[2] == res_str[3]:
```

Comment:
I see that you *are* using list slicing to check whether the "characters" of the numbers match, but you could go one step further and check the whole number at once by *reversing* the its string representation, i.e.:
```
prod_str = str(num1 * num2)
if prod_str[::-1] == prod_str:
    # prod_str[::-1] reverses the order of prod_str
    b=False
    break
```